package.json

```diff
@@ -11,6 +11,8 @@
     "download-locales": "node ./bin/update-translations",
     "test": "jest",
     "test-a11y": "node bin/lighthouse/index.js",
+    "i18n:extract": "node bin/extract-strings.mjs new-request-form 'src/modules/new-request-form/**/*.{ts,tsx}' src/modules/new-request-form/translations",
+    "i18n:update-translations": "node bin/update-module-translations.mjs new-request-form src/modules/new-request-form/translations/locales",
```

Comment:
At the moment these scripts are referring only to the `new-request-form`. The setup can be revisited later when we'll have more components